codice/ddf
# DDF-6168 Endorse saaj-api and activation-api ServiceMix spec bundles

#### What does this PR do?
Endorses ServiceMix saaj-api and activation-api bundles and exports their packages from bundle 0. This fixes an issue in Karaf causing linkage errors.

It also blacklists these bundles since their packages are already provided via the system bundle.

#### Who is reviewing it? 
@stustison @blen-desta @bakejeyner @garrettfreibott 

#### How should this be tested?
Ideally, setup a SOAP endpoint, send a request that causes a SOAPFaultException, and verify that you actually get the expected SOAPFault response back and that there are no linkage errors.



#### Any background context you want to provide?

#### What are the relevant tickets?
Fixes: https://github.com/codice/ddf/issues/6168

#### Screenshots


#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
